prow/tide/tide.go

```diff
@@ -437,7 +437,7 @@ func (c *Controller) Sync() error {
 	for pool := range poolChan {
 		pools = append(pools, pool)
 	}
-	sortPools(pools)
+	sortPools(pools, c.config().Tide.Priority)
```

Comment:
The only way in which the data we now sort differently is used is in serving tides status via http which is probably not want you want :)

Reply:
I _think_ what you intended to change was `pickSmallestPassingNumber` which is used to determine which PR to merge if there are multiple candidates and which PR to serially retest, if there are multiple candidates

Reply:
this time on the right method...